### ISSUE TITLE
Compatibility report: OrangepiOne

### ISSUE THREAD

**Issue by Aiki033** [shawly/docker-ps3netsrv#30]
**Device:**
OrangepiOne - Running Debian 11 + OpenMediaVault

**Status:**
[x] works - docker image v1.5.0 to v1.5.2 
[ ] not working

**Additional steps to get it to work?**
- Only docker image v1.5.0 to v1.5.2 works perfectly with my SBC, anything above these versions doesn't work. I can create a docker container for v1.6 up to the latest docker image but the problem is mounting a game. It takes forever to mount and once mounted, it throws and error message. Already tried both webmanMOD and MultiMan both latest on a latest 4.90 CFW.

**Did you verify that the standalone binaries of ps3netsrv work?**
[ ] yes
[x] no

**Have you read the troubleshooting section in the README?**
[x] yes
[ ] no

**Comment by Aiki033** [comment]
just an inquiry. Is there a disadvantage to not using the latest image or I should just stick to the working docker image?

**Comment by shawly** [comment]
Did you also try the standalone version of ps3netsrv? I wonder if it is actually related to my docker image or if it is just the latest builds of ps3netsrv.

**Comment by Aiki033** [comment]
honestly I'm ultra noob with linux just a few weeks learning  and I don't know how to compile this with my hardware. but I'll give a try  and report back here.

**Comment by Aiki033** [comment]
![trial 2](https://github.com/shawly/docker-ps3netsrv/assets/125640202/869bef69-0858-4d14-90e9-b13d8f553f90)
![Untitled](https://github.com/shawly/docker-ps3netsrv/assets/125640202/2d84beea-aef3-4d0c-89e4-f73a1f88ba25)

was able to compile latest stand alone build, I can confirm the same error appeared, same as with your latest docker image.
![IMG20231104114704](https://github.com/shawly/docker-ps3netsrv/assets/125640202/53151f6e-b72a-4ca6-8b3c-8c2bddaba42c)

after reverting to the working docker image, the game started working again.

**Comment by shawly** [comment]
Yeah in that case I can't fix the issue and it isn't related to my docker image, so you'd need to create an issue on the Webman Mod Github.

**Comment by Aiki033** [comment]
your docker image is working for me and I think that is enough. I'll try to post to Webman Mod Github nex time.

thank you.
